workflowparsers/phonopy/calculator.py

```diff
@@ -16,8 +16,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+from itertools import combinations
 import numpy as np
 import re
+from scipy.spatial.transform import Rotation as R
```

Comment:
is this being used? Impossible to grep if it is the case, sorry.

Reply:
No, it was from an older solution. Thx for spotting it!
Btw, this quite useful for vector-defined rotations. Definitely worthwhile remembering.